content/en/serverless/step_functions/installation.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Install Serverless Monitoring for AWS Step Functions
+kind: documentation
+further_reading:
+    - link: '/serverless/configuration/'
+      tag: 'Documentation'
+      text: 'Configure Serverless Monitoring'
+    - link: "/integrations/amazon_lambda/"
+      tag: "Documentation"
+      text: "AWS Lambda Integration"
+---
+
+### Requirements
+* The full Step Function execution length must be less than 15 minutes for full traces.
+* Linked Lambda traces are supported for Node.js (layer v94+) and Python (layer v75+) runtimes.
+
+### Setup
+
+{{< tabs >}}
+{{% tab "Serverless Framework" %}}
+
+For developers using [Serverless Framework][4] to deploy serverless applications, use the Datadog Serverless Framework Plugin.
+
+1. If you have not already, install the [Datadog Serverless Framework Plugin][1] v5.40.0+:
+
+    ```shell
+    serverless plugin install --name serverless-plugin-datadog
+    ```
+
+2. Ensure you have deployed the [Datadog Lambda Forwarder][2], a Lambda function that ships logs from AWS to Datadog, and that you are using v3.66.0+. You may need to [update your Forwarder][5].
+
+   Take note of your Forwarder's ARN.
+
+3. Enable all logging for your Step Function. In your AWS console, open your state machine. Click *Edit* and find the Logging section. There, set *Log level* to `ALL` and enable the *Include execution data* checkbox.
+   {{< img src="serverless/step_functions/aws_log.png" alt="AWS UI, Logging section, showing log level set to ALL." style="width:100%;" >}}
```

Comment:
Sorry this change for datadog-ci and plugin were only released yesterday. Step 3 is not needed for CI and Plugin because that would be automatic for both.

<img width="942" alt="image" src="https://github.com/DataDog/documentation/assets/47579703/fd9f66bf-ddb9-427d-8110-ba912772c831">

Reply:
- For Plugin, `enableStepFunctionTracing=true` does step 3 automatically. 
- For CI, the instrument command does this.